core/sodasql/cli/ingest.py

```diff
@@ -232,13 +342,35 @@ def ingest(
     if not soda_server_client.api_key_id or not soda_server_client.api_key_secret:
         raise ValueError("Missing Soda cloud api key id and/or secret.")
 
-    if tool == 'dbt':
-        dbt_manifest, dbt_run_results = resolve_artifacts_paths(
-            dbt_artifacts=dbt_artifacts,
-            dbt_manifest=dbt_manifest,
-            dbt_run_results=dbt_run_results
-        )
-        test_results_iterator = map_dbt_test_results_iterator(dbt_manifest, dbt_run_results)
+    if tool == "dbt":
+        if (
+            dbt_artifacts is not None
+            or dbt_manifest is not None
+            or dbt_run_results is not None
+        ):
+            manifest, run_results = load_dbt_artifacts(
+                dbt_artifacts,
+                dbt_manifest,
+                dbt_run_results,
+            )
+        else:
+            if dbt_cloud_api_token is None:
+                raise ValueError(
+                    f"Expecting an DBT cloud api token. Currently, dbt_cloud_api_token={dbt_cloud_api_token}."
+                )
+            elif dbt_cloud_account_id is None:
+                raise ValueError(
+                    f"Expecting an DBT cloud api token. Currently, dbt_cloud_account_id={dbt_cloud_account_id}."
```

Comment:
`Expecting dbt_cloud_account_id` , I don't think Currently is needed, maybe just rewrite to `dbt_cloud_account_id not provided`

Reply:
done